aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,53 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## November 2022
+
+- **Avatar**
+  - We've added Avatar [guidance and examples](https://design-system.hpe.design/components/avatar) and [Figma component](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1&t=bQK90xgXcu9vlYYm-0).
+- **Wizard**
+  - We've added Wizard [Anatomy](https://design-system.hpe.design/templates/wizard#anatomy) guidance.
+
+## October 2022
+
+- **HPE Brand Refresh**
+  - We've published a new [grommet-theme-hpe](https://www.npmjs.com/package/grommet-theme-hpe) with the latest HPE brand styling.
+  - We've added [Distinctive Brand Assets](https://design-system.hpe.design/foundation/distinctive-brand-assets) guidance and examples.
+  - We've added a extra Button kinds for 'cta-primary' and 'cta-alternate' along with [guidance](https://design-system.hpe.design/components/button#cta-button) for using them.
+  - We've clarified how to use the [Call-to-action Button](https://design-system.hpe.design/components/button#when-to-use-a-cta-button).
+  - We've updated the site to follow the HPE capitalization guidance for using mostly sentence case.
```

Comment:
should we link the guide?

Reply:
Good idea! I couldn't find anything for this though. I searched the site for "capitalization" and "sentence case" but didn't get any matches for them.

Reply:
https://design-system.hpe.design/foundation/voice-and-tone#capitalization

i wonder if we need to update our tags to find it

Reply:
Done